### ISSUE TITLE
Crash after busy

### ISSUE THREAD

**Issue by woessmich** [smarthausleben/ioBroker.wamo#7]
I experience some nightly crashes of the adapter even the device is not offline.
It starts with a couple of busy messages, the  this:

```
wamo.0 | 2022-06-16 03:19:44.106 | warn | [onUnload(callback)] was hit
-- | -- | -- | --
wamo.0 | 2022-06-16 03:19:44.103 | error | undefined
wamo.0 | 2022-06-16 03:19:44.102 | error | unhandled promise rejection: undefined
wamo.0 | 2022-06-16 03:19:44.102 | error | Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
wamo.0 | 2022-06-16 03:19:44.101 | info | timeout of 10000ms exceeded
wamo.0 | 2022-06-16 03:19:36.232 | warn | [async getData(statesToGet)] Device interface is bussy!
```

After a couple of days, iobroker does no longer restart the adapter because of the multiple failures.

**Comment by ACDelta** [comment]
Same problem here :-(

**Comment by smarthausleben** [comment]
Related to #1

**Comment by smarthausleben** [comment]
fixed in release v0.1.5
